package.json

```diff
@@ -1,5 +1,14 @@
 {
-  "build:global": "skate",
+  "global": "skate",
```

Comment:
For build purposes. This was more of a quick fix. I want to abstract out the blocks we've used and make them more configurable. I'd like to use them in other repos if possible. I'm trying other things than trying to keep skatejs-build up to date. Overriding configs has always been messy and I'm hoping that consuming building blocks that let you compose configs helps. Karma is a mess, though. I hate that config file. I'm hoping we can get wct running some day as it's a bit simpler to set up and use.